src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -208,6 +208,14 @@ public void stopModules(){
         backLeft.stop();
         backRight.stop();
     }
+
+    public void parkModules(){
```

Comment:
Write a comment that describes what it means to park modules, so others know what this method is supposed to do